lms/djangoapps/teams/tests/test_serializers.py

```diff
@@ -79,9 +79,9 @@ class TopicSerializerTestCase(SerializerTestCase):
     def test_topic_with_no_team_count(self):
         """
         Verifies that the `TopicSerializer` correctly displays a topic with a
-        team count of 0, and that it only takes one SQL query.
+        team count of 0, and that it takes a known number of SQL queries.
         """
-        with self.assertNumQueries(1):
+        with self.assertNumQueries(2):
```

Comment:
Why two? Where's the additional query coming from?

Reply:
Adding the modulestore lookup introduces an additional query.

A fix for this would be to add public/private info to the Django model which would remove the need for a modulestore query and allow us to filter the elasticsearch queries w/out an additional level of filtering.

Reply:
Just out of curiosity, could you run the tests with this as a 1 so it will fail and print out the queries, and copy and paste them here?

Reply:
Sure thing!

What I believe to be the `modulestore` lookup:
```
SELECT "xblock_django_xblockconfiguration"."id", 
"xblock_django_xblockconfiguration"."change_date", 
"xblock_django_xblockconfiguration"."changed_by_id", 
"xblock_django_xblockconfiguration"."enabled", 
"xblock_django_xblockconfiguration"."name", 
"xblock_django_xblockconfiguration"."deprecated",
1 AS "is_active"
FROM "xblock_django_xblockconfiguration"
WHERE ("xblock_django_xblockconfiguration"."id" IN (SELECT MAX(U0."id") AS "max"
FROM "xblock_django_xblockconfiguration" U0 GROUP BY U0."name")
AND "xblock_django_xblockconfiguration"."enabled" = 0)
ORDER BY "xblock_django_xblockconfiguration"."change_date" DESC
```

What I believe to be the team count lookup:
```
SELECT COUNT(*) AS "__count" FROM "teams_courseteam"
WHERE ("teams_courseteam"."course_id" = 'org.7/course_7/Run_7'
AND "teams_courseteam"."organization_protected" = 0
AND "teams_courseteam"."topic_id" = '0')
```

Reply:
I ask because I'm fairly sure modulestore lookups are from mongo, not mysql. I have no clue what xblock_django_xblockconfiguration is or what it's used for, but you're right, it's probably called under the hood somewhere when we do that modulestore lookup